src/common/filesystembase.cpp

```diff
@@ -480,6 +480,11 @@ bool FileSystem::isLnkFile(const QString &filename)
     return filename.endsWith(QLatin1String(".lnk"));
 }
 
+bool FileSystem::isExcludeFile(const QString &filename)
+{
+    return filename.endsWith(QLatin1String(".sync-exclude.lst"));
```

Comment:
I guess @mgallien will say better use `QStringLiteral` :)

You may also want to add `exclude.lst` as a exclude file. See `src/libsync/conifgfile.cpp:362`. Also it might be a good idea to create a named constant for these filenames.

Reply:
@FlexW 

> You may also want to add `exclude.lst` as a exclude file. See `src/libsync/conifgfile.cpp:362`.

Good idea.

Reply:
@FlexW 
> Also it might be a good idea to create a named constant for these filenames.

Could do, but, where should I put that constant?

Reply:
@allexzander How about `src/csync/csync_exclude.h`? May not be optimal but thats were it originally comes from I guess. `src/common/filesystembase.h` might also be a fit.

Reply:
@FlexW I've checked both suggestions and neither suits well. `src/common/filesystembase.h` is not really a good choice because it is a helper file that is not supposed to store implementation details of the sync engine.  `src/csync/csync_exclude.h` is also not the best place considering I will have then to import a part of sync engine into a helper class `src/common/filesystembase.h` which is in a `common` folder. I don't like having cross-dependencies. What I could do is - to add a separate header into a `common` folder. That header would then contain all the common constants. The problem is, however, I already have that header in another PR and don't want to create conflicts later. So, what I am thinking is - I should keep these file names hard-coded for now, and then, factor them out into a separate common header later in a separate PR.

Reply:
@allexzander At least you put sync engine detail functions into `src/common/filesystembase.h` with `isExclude()`. So I don't see why adding these constants here might be not a fit :)

Reply:
@FlexW You mean just adding constants into `filesystembase` and then using them just there? I've been thinking about something more common, like, having a header file that I can include into `filesystembase` and other places where this same `.sync-exclude.lst` is encountered, to make sure we have these constants all in one place.